WordPress/src/main/java/org/wordpress/android/ui/prefs/notifications/NotificationsSettingsFragment.java

```diff
@@ -126,6 +128,7 @@ public void onCreate(Bundle savedInstanceState) {
         addPreferencesFromResource(R.xml.notifications_settings);
         setHasOptionsMenu(true);
         removeSightAndSoundsForAPI26();
+        if (mBuildConfigWrapper.isJetpackApp()) removeFollowedBlogsPreference();
```

Comment:
Suggestion (💡): I would have considered keeping this consistent with the above method, calling it a `removeFollowedBlogsForJetpack()` instead and including the `if/else` logic within, just like it happens above. This is purely for consistency purposes, maybe slightly for readability, but not sure. Feel free to ignore if you feel otherwise or you find it super minor.

Reply:
I was actually toying with this idea, and I then reverted to the present form as calling the method for the WordPress app was not required and we pre-checked it by putting the if check.  I will go ahead and move the condition inside the method as it came to my mind as well, and it is brought up in this discussion.

Reply:
Awesome, many thanks Ashita! 🚀